src/main/resources/db/migration/V10__tontrinnstatus.sql

```diff
@@ -0,0 +1,13 @@
+CREATE table Totrinnsstatus
```

Comment:
Denne burde kanskje få komme med i PR som legger inn støtte for databasen? Eks hvis den ikke passer/annet